### PR TITLE
Objects created by the staff of animation eventually revert if there's no one to attack

### LINKED ISSUE #4490: Staff of animation is overpowered
Some objects (e.g. the ion rifle) have too much health and do too much damage. Especially considering the amount of other hostile objects on the station.

This has been the case since it was added. Issue report #286 ported from Dec 2012 is the earliest mention of the staff I've found. So it's been a problem for nearly two years.

### LINKED ISSUE #13835: Staff of Animation is annoying
You can animate an infinite number of things at once, and they don't revert until you kill them. Given enough time the Wizard can have hundreds of objects animated throughout the station, without any real crew interaction.

### PR DESCRIPTION
**Pull request by Incoming5643**
Offered as an alternative to #13839

Fixes #4490
Fixes #13835

### CODE REVIEW

**Comment by tkdrg** [comment]
I like this one more.

**Comment by RemieRichards** [comment]
might want to check AI status too.

**Comment by xxalpha** [comment]
:-1:

I like the other solution better.

**Comment by KorPhaeron** [comment]
I like this solution and have closed mine

**Comment by newfren** [comment]
Doesn't this already happen? Fund it if it doesn't.

File: code/modules/mob/living/simple_animal/hostile/mimic.dm [hunk 1]
```diff
@@ -119,6 +119,8 @@ var/global/list/protected_objects = list(/obj/structure/table, /obj/structure/ca
 
 /mob/living/simple_animal/hostile/mimic/copy/Life()
 	..()
+	if(!target && !client) //Objects eventually revert to normal if no one is around to terrorize
```
Comment (MrStonedOne) [inline code/modules/mob/living/simple_animal/hostile/mimic.dm]: You should have it checking key or ckey, as getting disconnected either doesn't, or shouldn't cause it to start attacking again.

**Comment by duncathan** [comment]
@Incoming5643 do the mso

**Pull request merged**